File: modulo1/segmento2/apunte1/ejercicio1.web/ejercicio1.js
function saludar(){
	
	const barrio = prompt('Ingrese su barrio');
	const direccion = prompt('Ingrese la direccion de la comisaria mas cercana: ');
	const entrecalle1 = prompt('Ingrese entrecalle1: ');
	const entrecalle2 = prompt('Ingrese entrecalle2: ');
	
	alert(`La comisaria de  ${barrio} esta en la direccion ${direccion} entre la calle ${entrecalle1} y ${entrecalle2}`);
}

File: modulo1/segmento2/apunte1/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		String barrio;
		System.out.print("Ingrese su barrio: ");
		barrio = teclado.nextLine();
		
		String direccion;
		System.out.print("Ingrese la direccion de la comisaria mas cercana: ");
		direccion= teclado.nextLine();

		String entrecalle1;
		System.out.print("Entrecalle1: ");
		entrecalle1= teclado.nextLine();

		String entrecalle2;
		System.out.print("Entrecalle2: ");
		entrecalle2= teclado.nextLine();

		System.out.println("La comisaria de " + barrio + " se encuentra en " + direccion + " entre las calles " + entrecalle1 + " y " + entrecalle2);

	}
}
